src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -862,4 +862,38 @@
           }
         }
       }
+    },
+    "/admin/redpanda-services/restart": {
```

Comment:
Should this go on the admin API (usually at `:9644`)?

Reply:
I agree that is a better place for this method, but switching over to the admin API may take more time so I'll stick with the current location for now. I'll create a ticket to tackle this as a follow-up.

Reply:
I'd prefer not moving the endpoints around, as that will involve deprecation. You may not have access to the `api` object from a handler, and referring to the earlier comment, it will likely invalidate much of what's in the handler.

Reply:
Done.